docs/tutorial/queries.rst

```diff
@@ -245,11 +243,11 @@ the ``first_name`` and ``last_name`` properties. This time we will use
     .........         }
     .........     }
     ......... };
-    CREATE MIGRATION
-    tutorial> COMMIT MIGRATION movies;
+    START MIGRATION
+    tutorial> POPULATE MIGRATION;
+    POPULATE MIGRATION
+    tutorial> COMMIT MIGRATION;
```

Comment:
Can COMMIT MIGRATION automatically call POPULATE MIGRATION if it wasn't called before? That would simplify the initial manual schema creation in the tutorial(s) (although while complicating the spec a bit)

Reply:
We should be using the CLI in the tutorials.  All `MIGRATION` commands are now considered low-level plumbing and we shouldn't encourage newcomers to use them or make them deceptively easy.